docker-compose.yml

```diff
@@ -19,28 +19,82 @@ services:
       - "26667:26667"
 
   # The Tendermint node
-  tendermint:
+  tendermint-node0:
     image: "tendermint/tendermint:latest"
-    container_name: tendermint
+    container_name: tendermint-node0
     ports:
       - "26656:26656"
       - "26657:26657"
     volumes:
-      - "tendermint_data:/tendermint"
+      - ~/scratch/testnet_build/node0:/tendermint:Z
     command: start --proxy-app=tcp://pd:26658
+    environment:
+      - ID=0
+      - LOG=${LOG:-tendermint.log}
     depends_on:
       - pd
     networks:
       localnet:
         ipv4_address: 192.167.10.2
 
+  # The Tendermint node
+  # tendermint-node1:
+  #   image: "tendermint/tendermint:latest"
+  #   container_name: tendermint-node1
+  #   ports:
+  #     - "26659-26660:26656-26657"
+  #   volumes:
+  #     - ~/scratch/testnet_build/node1:/tendermint:Z
+  #   command: start --proxy-app=tcp://pd:26658
+  #   environment:
+  #     - ID=1
+  #     - LOG=${LOG:-tendermint.log}
+  #   depends_on:
+  #     - pd
+  #   networks:
+  #     localnet:
+  #       ipv4_address: 192.167.10.3
+
+  # # The Tendermint node
+  # tendermint-node2:
+  #   image: "tendermint/tendermint:latest"
+  #   container_name: tendermint-node2
+  #   ports:
+  #     - "26661-26662:26656-26657"
+  #   volumes:
+  #     - ~/scratch/testnet_build/node2:/tendermint:Z
+  #   command: start --proxy-app=tcp://pd:26658
+  #   environment:
+  #     - ID=2
+  #     - LOG=${LOG:-tendermint.log}
+  #   depends_on:
+  #     - pd
+  #   networks:
+  #     localnet:
+  #       ipv4_address: 192.167.10.4
+
+  # # The Tendermint node
+  # tendermint-node3:
+  #   image: "tendermint/tendermint:latest"
+  #   container_name: tendermint-node3
+  #   ports:
+  #     - "26663-26664:26656-26657"
+  #   volumes:
+  #     - ~/scratch/testnet_build/node3:/tendermint:Z
+  #   command: start --proxy-app=tcp://pd:26658
+  #   environment:
+  #     - ID=3
+  #     - LOG=${LOG:-tendermint.log}
+  #   depends_on:
+  #     - pd
+  #   networks:
+  #     localnet:
+  #       ipv4_address: 192.167.10.5
+
```

Comment:
The idea would be that these could be uncommented to run a multi-node testnet, right (mod having to run separate `pd` and `postgres` instances)?

Reply:
Exactly